.github/workflows/create_nightly_release.yml

```diff
@@ -28,25 +46,35 @@ jobs:
       - name: Make build folder
         run: mkdir ${{ github.workspace }}/build
       - name: Run the Docker image
-        run: docker run -i -v ${{ github.workspace }}:/havoc portapack-dev
+        run: docker run -e VERSION_STRING=${{ steps.version_date.outputs.date }} -i -v ${{ github.workspace }}:/havoc portapack-dev
       - name: Create Firmware ZIP
         run: |
           zip --junk-paths firmware build/firmware/portapack-h1_h2-mayhem.bin
       - name: Create SD Card ZIP
         run: |
           zip -r sdcard.zip sdcard
+      - name: Create changelog
+        run: |
+          CHANGELOG=$(git log next --since="24 hours" --pretty=format:"- %h - @%an: %s")
```

Comment:
this will generate invalid users

Reply:
How so?

Reply:
Ignore me, I saw you comment. Have fixed in https://github.com/eried/portapack-mayhem/pull/534